lib/public/views/Runs/Overview/RunsOverviewPage.js

```diff
@@ -57,3 +59,23 @@ const exportRunsButton = ({ runsOverviewModel, modalModel }) => h('button.btn.bt
     disabled: (runsOverviewModel.runs?.payload?.length ?? 0) === 0,
     onclick: () => modalModel.display({ content: (modalModel) => exportRunsModal(runsOverviewModel, modalModel), size: 'medium' }),
 }, 'Export Runs');
+
+/**
+ * Builds a button which will toggle the physics filter
+ * @param {OverviewModel} runsOverviewModel the model of the runs overview
+ * @returns {vnode} with button
+ */
+const showPhysicsButton = (runsOverviewModel) => {
+    const isPhysicsActive = runsOverviewModel.isDefinitionInFilter(RunDefinition.Physics);
+    const buttonColor = isPhysicsActive ? 'primary' : 'secondary';
+
+    return h(`button.btn.btn-${buttonColor}.w-10.h2.ml2#show-physics-trigger`, {
```

Comment:
Avoid as much as possible to put fixed width/height on your components, especially on components that have a fixed size (such as a button).
Same for margin. When you just want spacing between items, please use gap (we have classes for it, it's `g1`, `g2` and `g3`) on the flex parent. Doing so, please also remove the padding of the run number input

Reply:
Fixed this issue, the only problem now is a layout shift when the text changes to show all, so there are a few solutions to this.

1. We only show the text Show PHYSICS and just make it blue when it is active and grey when not
2. We do use the change in text and thus use a fixed with for the button
3. We make it a checkbox

I let the decision be yours, for now I did not switch the text and this chose for option 1